### DISCUSSION TITLE
THERE IS A BUG, BIG BUG

### DISCUSSION THREAD

**Discussion by viponedream** [swar/Swar-Chia-Plot-Manager#612] [Q&A]
I SET 3 max_concurrent, BUT IT RUNS IN 5 TO FULL THE DRIVE. 

![QQ图片20210520104336](https://user-images.githubusercontent.com/16624529/118911404-b8958780-b958-11eb-80aa-628fc24fafb2.png)

`  - name: E3-SSD-1
    max_plots: 999

    farmer_public_key: 
    pool_public_key: 
    temporary_directory: G:\Plotter
    temporary2_directory:
    destination_directory: F:\FARM

    size: 32
    bitfield: true
    threads: 3
    buckets: 64
    memory_buffer: 5000
    max_concurrent: 3
    max_concurrent_with_start_early: 4
    initial_delay_minutes: 0
    stagger_minutes: 60
    max_for_phase_1: 2
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
    exclude_final_directory: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: false
    cpu_affinity: [ 0, 1, 2, 3, 4, 5 ]

`

**Comment by swar** [comment]
I don't see the bug. It's running 3 concurrent on that job. What is your global max_concurrent set too? @viponedream

**Reply by viponedream** [reply]
it's running 6 concurrent on that job. but i just set 3.

**Reply by viponedream** [reply]
i only have 1 job

**Reply by Drummas** [reply]
Your temp drive is full and those jobs crashed, hence why you have 6 instead of 3

**Comment by viponedream** [comment]
1, before start a new job,   can it calculate the free tmp space  first?  
2, when it finished,  more job try to copy to the same destination_directory at the same time ,  could it do it one by one?
